package.json

```
@@ -74,47 +74,47 @@
+    "log:single": "log() { echo \\\\x1B[34m--$1\\\\x1B[0m; }; log",
     "log:header": "log() { echo '============='; echo $1; echo '============='; }; log",
-    "log:comp": "log() { echo \\\\e[32m$1\\\\e[0m; }; log",
-    "=============================================================================== CJS": "",
-    "copy:cjs:components": "pnpm copyfiles -u 2 -e 'components/mjs/**/*.js' 'components/mjs/**/*' components/cjs",
-    "clean:cjs:bundle": "pnpm clean:dir bundle-cjs",
-    "clean:cjs:components": "pnpm -s log:single \"Cleaning cjs components\" pnpm rimraf components/cjs",
-    "compile:cjs:components": "pnpm -s log:single 'Compiling component files'; pnpm tsc --project tsconfig/components.json",
-    "finalize:cjs:components": "pnpm -s log:comp 'Finalize cjs components'; pnpm -s copy:cjs:components && pnpm -s copy:pkg components/cjs && pnpm -s clean:lib cjs",
-    "compile:cjs": "pnpm -s log:single 'Compiling cjs typescript files' && pnpm tsc --project tsconfig/cjs.json",
-    "finalize:cjs:bundle": "pnpm -s log:single 'Finalize cjs bundle'; echo '{\n  \"type\": \"commonjs\"\n}' > bundle-cjs/package.json;",
-    "make:cjs:components": "make() { pnpm -s log:single 'Making cjs components'; components/bin/makeAll --cjs --terse --bundle-cjs $1 components/cjs; }; make",
-    "build:cjs:components:src": "pnpm -s log:comp 'Building cjs components sources'; pnpm clean:cjs:components && pnpm compile:cjs:components && pnpm finalize:cjs:components",
-    "build:cjs:components": "pnpm -s log:comp 'Building cjs components'; pnpm -s build:cjs:components:src && pnpm clean:dir bundle-cjs && pnpm -s make:cjs:components --build",
-    "build:cjs:src": "pnpm -s log:comp 'Building cjs sources'; pnpm -s link:full && pnpm clean:dir cjs && pnpm -s compile:cjs && pnpm -s copy:assets cjs && pnpm -s copy:pkg cjs",
-    "build:cjs": "pnpm -s log:header 'Building cjs'; pnpm -s build:cjs:src && pnpm -s build:cjs:components",
-    "=============================================================================== MJS": "",
-    "make:mjs:components": "pnpm -s log:single 'Making mjs components'; components/bin/makeAll --mjs --terse components/mjs",
-    "finalize:mjs:bundle": "pnpm -s log:single 'Finalize mjs bundle'; echo '{\n  \"type\": \"commonjs\"\n}' > bundle/package.json;",
-    "compile:mjs": "pnpm -s log:single 'Compiling mjs typescript files'; pnpm tsc --project tsconfig/mjs.json",
-    "build:mjs:components": "pnpm -s log:comp 'Compiling mjs component files'; pnpm clean:lib mjs && pnpm clean:dir bundle && pnpm make:mjs:components && pnpm finalize:mjs:bundle",
-    "build:mjs:src": "pnpm -s log:comp 'Building mjs sources'; pnpm -s link:full && pnpm -s clean:dir mjs && pnpm -s compile:mjs && pnpm -s copy:assets mjs",
-    "build:mjs": "pnpm -s log:header 'Building mjs'; pnpm -s build:mjs:src && pnpm -s build:mjs:components",
+    "log:comp": "log() { echo \\\\x1B[32m$1\\x1B[0m\\; }; log",
```

Superfluous `\\;` leads to errors. 
```suggestion
    "log:comp": "log() { echo \\\\x1B[32m$1\\x1B[0m; }; log",
```